### ISSUE TITLE
making screenshots stopped working on windows

### ISSUE THREAD

**Issue by RubendeBruin** [marcomusy/vedo#395]
Making a screenshot using a background renderer is not working on windows 10.

vedo version: 2021.0.2
vtk version: 9.0.1

```python
import vedo

c = vedo.Cone()

camera = dict()
camera['pos'] = (10,0,0)
camera['focalPoint'] = (0,0,0)

# display on screen: works
p = vedo.Plotter(bg=(0.5,0.5,0.9))
p.add(c)
p.show(camera=camera)

# render offscreen: actor is not displayed
b = vedo.Plotter(offscreen=True, bg=(0.5,0.5,0.9))
b.add(c)
b.show(camera=camera)

# render and display using PIL image, saving to disk gives same result
npdata = b.screenshot(returnNumpy=True)

from PIL import Image
image = Image.fromarray(npdata)
image.show()
```

Expecting:
![image](https://user-images.githubusercontent.com/34062862/118381168-c080bf00-b61a-11eb-8cd3-df8ac6b67f11.png)

but getting:
![image](https://user-images.githubusercontent.com/34062862/118381175-c70f3680-b61a-11eb-9aae-d1579ccef897.png)

**Comment by RubendeBruin** [comment]
With vedo 2020.4.2:

![image](https://user-images.githubusercontent.com/34062862/118381256-a8f60600-b61b-11eb-80e3-998d469ae901.png)

**Comment by RubendeBruin** [comment]
My mistake, solved in 2021.0.3
